### PR TITLE
Update elixir and erlang versions

### PR DESCRIPTION
**Pull request by eric-hostalery**

### CODE REVIEW

File: elixir/Dockerfile [hunk 1]
```diff
@@ -4,10 +4,10 @@ LABEL vendor="JobTeaser"
 LABEL maintainer="opensource@jobteaser.com"
 
 # Erlang Solutions build
-ENV ERLANG_VERSION 25.3
+ENV ERLANG_VERSION 26.1.2
 
-ENV ELIXIR_VERSION 1.14.3
-ENV NODE_VERSION 14.15.4
+ENV ELIXIR_VERSION 1.16.0
+ENV NODE_VERSION 18.12.0
```
Comment (sebn) [inline elixir/Dockerfile:10]: Was nodejs bump required to bump the other 2?
(just asking because it may require a more thorough FR on the UI side...)

Comment (sebn) [reply elixir/Dockerfile:10]: (also latest LTS is 20.11.0, but it's possible that some of our internal JS deps require a lower version...)

**Review by sebn — APPROVED** [review]
(no blocker anyway) :+1:

File: elixir/Dockerfile (hunk 1 above)
Comment (eric-hostalery) [reply elixir/Dockerfile:10]: We bumped the nodejs version in `.tools-version` to 18.12.0 a moment ago, it was just to make it consistent.
 Indeed, we could upgrade to 20.11.0, but I'm not confident with doing it right now, we need to do more tests before

**Pull request merged**